### ISSUE TITLE
Save as: save button should have visible focus.

### ISSUE THREAD

**Issue by VirginiaBalseiro** [linkeddata/dokieli#503]
The green interactive button (‘Save’ button) element doesn’t have a visible focus (highlighted) when it receives focus using the keyboard.

See: https://www.w3.org/WAI/WCAG22/Understanding/focus-visible
